tools/checked-c-convert/Utils.cpp

```diff
@@ -120,12 +120,11 @@ bool isDeclarationParam(clang::Decl *param) {
 
 static std::string storageClassToString(StorageClass SC) {
   switch (SC) {
+    default: return "";
```

Comment:
[Nit] Can we move the default after the case labels?

Reply:
Sorry, didn't notice this is merged already. Never mind :)

Reply:
> [Nit] Can we move the default after the case labels?

For better readability, LLVM convention is to have default as the first label in a switch-case.